data/experiment-recipe-samples/rollout-mobile.json

```diff
@@ -0,0 +1,38 @@
+{
+  "schemaVersion": "0.1.0",
+  "id": "rollout-mobile-example",
+  "slug": "rollout-mobile-example",
+  "application": "reference_browser",
```

Comment:
I was expecting this to be specific, but from what I understand from @jhugman, the value of the application is redundant